### ISSUE TITLE
No credentials error when using `allowCredentials`

### ISSUE THREAD

**Issue by erickriva** [peterferguson/react-native-passkeys#26]
Testing example app on Android, I tried to create a passkey and it worked, but then trying to authenticate with `allowCredentials` a `No credentials` error is shown.

**Passkey creation response:**
```
{"id": "8iF-k1mOuwOzHekbiJTOlw", "rawId": "8iF-k1mOuwOzHekbiJTOlw", "type": "public-key"}
```

**Credentials used to start authentication:**
```
allowCredentials: [{ id: "8iF-k1mOuwOzHekbiJTOlw", type: "public-key" }]
```

**Comment by peterferguson** [comment]
Interesting do you have a repro? 

This exact thing is done in the example and last i checked it worked

see 
```ts
const readBlob = async () => {
		const json = await passkey.get({
			rpId: rp.id,
			challenge,
			extensions: { largeBlob: { read: true } },
			...(credentialId && {
				allowCredentials: [{ id: credentialId, type: "public-key" }],
			}),
		});


		console.log("read blob json -", json);


		const blob = json?.clientExtensionResults?.largeBlob?.blob;
		if (blob) alert("This passkey has blob", base64UrlToString(blob));


		setResult(json);
	};
```

**Comment by peterferguson** [comment]
could have been related to [this](https://github.com/peterferguson/react-native-passkeys/pull/25/files)

**Comment by yagopv** [comment]
I can confirm that I had the same issue and that it was fixed with 0.3.1 :D :D
